src/Components/SingleArticle.jsx

```diff
@@ -20,14 +21,20 @@ useEffect(() => {
 
       if (isLoading) return <p>Loading...</p>;
       
-      return (
+      return (<>
```

Comment:
I'd like to see this as a more semantic tag than a fragment, it you want to get technical it is an article and that is also a tag you can use 😄